ui/design-system-core/src/components/BasicCardBox/index.tsx

```diff
@@ -45,10 +47,10 @@ const BasicCardBox: React.FC<PropsWithChildren<IBasicCardBox>> = props => {
 
   const className = `flex flex-col shadow-${elevation} w-full ${pad ? pad : 'p-0'} ${
```

Comment:
here you can use the cx utility from twind: const class = cx(`flex flex-col....) and on line 53 just use `<div` className={class}>`